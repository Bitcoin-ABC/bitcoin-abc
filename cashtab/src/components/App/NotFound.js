// Copyright (c) 2024 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.

import React from 'react';
import styled from 'styled-components';

const Centered = styled.div`
    position: fixed;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    color: ${props => props.theme.contrast};
`;

const NotFound = () => (
    <Centered>
        <h1>Page not found</h1>
    </Centered>
);

export default NotFound;
