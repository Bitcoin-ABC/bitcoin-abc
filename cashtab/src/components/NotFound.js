// Copyright (c) 2024 The Bitcoin developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or http://www.opensource.org/licenses/mit-license.php.

import React from 'react';
import { Row, Col } from 'antd';

const NotFound = () => (
    <Row justify="center" type="flex">
        <Col span={8}>
            <h1>Page not found</h1>
        </Col>
    </Row>
);

export default NotFound;
