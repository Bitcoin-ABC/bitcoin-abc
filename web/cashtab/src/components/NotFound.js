import React from 'react';
import { Row, Col } from 'antd';

const NotFound = () => (
    <Row justify="center" type="flex">
        <Col span={8}>
            <h1>Page not found</h1>
        </Col>
    </Row>
);

export default NotFound;
