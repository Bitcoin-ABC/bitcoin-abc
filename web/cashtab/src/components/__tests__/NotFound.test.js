import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '@components/NotFound';

test('Render NotFound component', () => {
    const component = renderer.create(<NotFound />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
