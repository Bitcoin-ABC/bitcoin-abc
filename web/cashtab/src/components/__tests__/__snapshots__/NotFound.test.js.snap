// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotFound component 1`] = `
<div
  className="ant-row ant-row-center"
  style={Object {}}
  type="flex"
>
  <div
    className="ant-col ant-col-8"
    style={Object {}}
  >
    <h1>
      Page not found
    </h1>
  </div>
</div>
`;
